lens/lens_result.ml

```diff
@@ -12,6 +12,11 @@ let bind v ~f =
 let map v ~f =
   bind v ~f:(fun v -> f v |> return)
 
+let ok_exn v =
+  match v with
+  | Result.Ok v -> v
+  | _ -> failwith "Unexpected error unpacking result."
```

Comment:
Generally the `ok_exn` function should only be used for cases where it is absolutely certain that something cannot fail or if one has checked it before already. There is only a single hack in the actual code that uses this, all other uses are in the unit testing library. Unfortunately `lens` can't depend on `core` so using `Errors.InternalError` is not possible. In the long term I am trying to avoid exceptions though.

Reply:
Of course! I missed that. Thanks for the clarification.